lib/widgets/details.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+
+class Details extends StatelessWidget {
+  Details({
+    super.key,
+    required this.text,
+    required this.text2,
+    required this.text3,
+  });
+
+  String text;
+  String text2;
+  String text3;
```

Comment:
be more descriptive with vartiable naming.

also, make them all final:

```suggestion
  final String text;
  final String text2;
  final String text3;
```